nodesSetup.json

```diff
@@ -1,5 +1,5 @@
 {
-  "startTime": 1648551600,
+  "startTime": 1693317600,
```

Comment:
if we don't release it in time this needs to be changed.

Reply:
right, this is set for the dry-run